controllers/user.controller.go

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import (
+	"tiny-site-backend/models"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+func GetSelf(c *fiber.Ctx) error {
+	user := c.Locals("user").(models.UserResponse)
```

Comment:
can you add a inline comment, telling about from where do we get the user data in the context

Reply:
done